pkg/cmd/util/clientcmd/factory.go

```diff
@@ -323,7 +323,7 @@ func NewFactory(clientConfig kclientcmd.ClientConfig) *Factory {
 			var err error
 			var pods *api.PodList
 			for pods == nil || len(pods.Items) == 0 {
-				if t.LatestVersion == 0 {
+				if t.Status.LatestVersion == 0 {
 					time.Sleep(2 * time.Second)
```

Comment:
Ok, thought you were talking about Status here too. Opened https://github.com/openshift/origin/issues/6344